### PR TITLE
[ADD] purchase_reorder_control

### PR DESCRIPTION
**Pull request by trisdoan**

### CODE REVIEW

File: purchase_reorder_control/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Purchase Reorder Control",
+    "summary": "Restrict reordering unpurchasable product",
```
Comment (nilshamerlinck) [inline purchase_reorder_control/__manifest__.py]:
```suggestion
    "summary": "Restrict reordering unpurchaseable product",
```

File: purchase_reorder_control/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
```
Comment (nilshamerlinck) [inline purchase_reorder_control/models/product_template.py:4]: copyrights?

File: purchase_reorder_control/models/product_template.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class ProductTemplate(models.Model):
```
Comment (nilshamerlinck) [inline purchase_reorder_control/models/product_template.py:7]: don't we need to do it on variant as well?

**Review by nilshamerlinck — CHANGES_REQUESTED** [review]

**Comment by trisdoan** [comment]
Hi anh @nilshamerlinck, I updated as you suggest

**Review by nilshamerlinck — APPROVED** [review]

**Pull request closed without merging**